docs/guide/http_cache.md

```diff
@@ -494,8 +494,11 @@ Here are some generic recommendations on how to approach personalized content wi
 1\. ESI + Vary by Cookie:
 
 - As default VCL strips everything but session cookie this means this will effectively be cached "per user".
-   - If you are on single server setup with neither Varnish nor Fastly, you can do the same cookie logic on the Web server instead.
+    - If you are on single-server setup with neither Varnish nor Fastly, you can use the same cookie logic on the Web server instead.
```

Comment:
Possible suggestion to simplify this sentence:
```suggestion
    - With Symfony Proxy you'll need to implement cookie stripping on Web server _(Nginx/Apache)_ configuration
```
Notes:
- Possible extension to this sentence `, in order to avoid this varying on all possible analytics cookies and so on.`
- This is not possible on platform.sh, but we ship with Varnish by default there.
- It's technical possible to implement this by adapting `src/CacheKernel.php`, but that is not advisable in terms of upgradability.